Fix bugs to enable module to be usable as an npm module

andyzg opened the pull request:
To test:
- Install the module using `npm install @codeium/react-code-editor` in a React app or in the `demo/` folder of this folder
- Import CodeiumEditor from the module
- Default layout is 100% width and 300px height
- No branding changes or API restrictions were made
- Preview:

<img width="839" alt="image" src="https://github.com/Exafunction/codeium-react-code-editor/assets/3849459/0a32be30-4771-4d31-afdc-2874cd5a23af">

Remaining:
- There's a bug that happens upon completion. Needs more investigation
<img width="991" alt="image" src="https://github.com/Exafunction/codeium-react-code-editor/assets/3849459/5df8fa61-5dec-4f05-adeb-270239ed1095">

khou22 commented on demo/app/page.tsx:
```diff
@@ -6,7 +6,9 @@ import { CodeiumEditor } from "@codeium/react-code-editor";
   return (
     <main className={styles.main}>
-      <CodeiumEditor language="python" />
+      <div>
+        <CodeiumEditor language="python" />
+      </div>
```
Does it have to nested in a `div` now?

khou22 commented on src/components/CodeiumEditor/CodeiumEditor.tsx:
```diff
@@ -77,12 +77,20 @@ export const CodeiumEditor: React.FC<CodeiumEditorProps> = (props) => {
     defaultValue: getDefaultValue(props.language),
   };
 
+  const layout = {
```
Worth adding these usage caveats to the function docstring

andyzg commented:
@khou22 packed two fixes into the recent changes along with addressing comments:
- Catch network error for AcceptComplete
- Fix logo not showing if the Editor is not at the top

khou22 reviewed (approved):
Nice!

khou22 reviewed (approved):
Skimmed through, looks reasonable
